frontend/store/token.js

```diff
@@ -55,7 +55,19 @@ export const getters = {
         }
       })
       .reverse(),
-  distributionConfigs: state => state.distributionConfigs
+  distributionConfigs: state =>
+    state.distributionConfigs
+      .map(distribution => {
+        const name = accounts.find(account => distribution.lockedWallet.toLowerCase() === account.address.toLowerCase()).name
+        return {
+          lockedWallet: distribution.lockedWallet,
+          unlockWallet: distribution.unlockWallet,
+          ratioDTV: distribution.ratioDTV / Math.pow(10, distribution.ratioDecimals),
```

Comment:
You should not replace ratioDTV but add a new parameter.
We will need to access the "raw" ratioDTV for edition.

```suggestion
          ratioDTV: distribution.ratioDTV
          ratioDTVPercent: distribution.ratioDTV / Math.pow(10, distribution.ratioDecimals),
```